src/assets/stylesheets/2d-hud.scss

```diff
@@ -199,32 +199,58 @@
   }  
 }
 
-:local(.iconButton.spawn_pen) {
-  background-image: url(../hud/spawn_pen.png);
+:local(.iconButton.pen) {
+  background-image: url(../hud/pen_off.png);
 }
-:local(.iconButton.spawn_pen:hover) {
+:local(.iconButton.pen:hover) {
   @media (pointer: fine) {
-    background-image: url(../hud/spawn_pen-hover.png);
+    background-image: url(../hud/pen_off-hover.png);
   }
 
   @media (pointer: coarse) {
-    background-image: url(../hud/spawn_pen.png);
+    background-image: url(../hud/pen_off.png);
   }
 }
 
-:local(.iconButton.spawn_camera) {
-  background-image: url(../hud/spawn_camera.png);
+:local(.iconButton.pen.active) {
+  background-image: url(../hud/pen_on.png);
 }
-:local(.iconButton.spawn_camera:hover) {
+:local(.iconButton.pen.active:hover) {
   @media (pointer: fine) {
-    background-image: url(../hud/spawn_camera-hover.png);
+    background-image: url(../hud/pen_on-hover.png);
   }
 
   @media (pointer: coarse) {
-    background-image: url(../hud/spawn_camera.png);
+    background-image: url(../hud/pen_on.png);
+  }  
+}
+
+:local(.iconButton.camera) {
+  background-image: url(../hud/camera_off.png);
+}
+:local(.iconButton.camera:hover) {
+  @media (pointer: fine) {
+    background-image: url(../hud/camera_off-hover.png);
+  }
+
+  @media (pointer: coarse) {
```

Comment:
I don't really know what else we can do here -- the CSS selectors we are using I think are correct and the cascade is mitigating some browser issues. On the touchscreen laptop, you want to avoid showing the hover if they used their finger to click, otherwise show it if they used the mouse. I don't think we should be wiring up onHover functions, that seems like not worth the additional complexity.